pom.xml

```diff
@@ -17,19 +17,30 @@
     <version>5.2.0</version>
```

Comment:
I think we should have one consuming the release versions
then a follow-up 5.2.1-SNAPSHOT consuming the snapshots